app/components/DropDownButton.js

```diff
@@ -5,7 +5,7 @@ import { Button } from 'react-native-elements'
 import { MaterialIcons } from '@expo/vector-icons'
 import { ThemeContext } from '../theme-context'
 
-const DropDownButton = ({ title, onPress, }) => {
+const DropDownButton = ({ title, onPress, containerStyle }) => {
     const { theme } = useContext(ThemeContext)
     const s = getStyles(theme)
```

Comment:
If you wanted to retain the style object rather than having to use inline styles for the containerStyle, we could pass `containerStyle` here into `getStyles`.

Reply:
I barely remember why I did this. I think it was because there were some instances where I wanted a different margin but I still wanted all the other inline styles. I am for whatever is best practice. I think I used PbmButton as guidance here.